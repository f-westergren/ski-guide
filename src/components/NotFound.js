import React from 'react';
import { Container } from 'reactstrap';

export default function NotFound() {
  return (
    <Container>
      <h2 className="text-center mt-5">404 Page Not Found</h2>
    </Container>
  )
}