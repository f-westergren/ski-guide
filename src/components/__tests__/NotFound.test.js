import React from "react";
import { render } from "@testing-library/react";
import NotFound from "../NotFound";

it("renders without crashing", function() {
  render(<NotFound />);
});

it("matches snapshot", function() {
  const { asFragment } = render(<NotFound />);
  expect(asFragment()).toMatchSnapshot();
});
