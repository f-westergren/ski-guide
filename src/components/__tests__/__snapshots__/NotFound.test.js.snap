// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h2
      class="text-center mt-5"
    >
      404 Page Not Found
    </h2>
  </div>
</DocumentFragment>
`;
